qa/integration-tests/src/test/java/io/camunda/zeebe/broker/it/clustering/ClusteringRule.java

```diff
@@ -720,6 +721,14 @@ public SnapshotId waitForSnapshotAtBroker(final Broker broker) {
     return waitForNewSnapshotAtBroker(broker, null);
   }
 
+  public void takeSnapshot(final Broker broker) {
+    broker.getBrokerAdminService().takeSnapshot();
```

Comment:
I hope that also other snapshot method works (so without triggering it explicit :D )

Reply:
You mean by advancing the clock? Should we add a test using both - one that trigger explicitly and one that using the clock?

Reply:
I think this would be nice, since these are two different code paths :+1: